src/renderer/components/opds/BrowserResult.tsx

```diff
@@ -24,14 +24,20 @@ import MessageOpdBrowserResult from "./MessageOpdBrowserResult";
 
 interface BrowserResultProps extends RouteComponentProps, ApiProps, TranslatorProps {
     url: string;
-    result?: THttpGetOpdsResultView | string;
+    result?: THttpGetOpdsResultView;
```

Comment:
@panaC what do you think of this code change?

Reply:
result may technically contains an string error message.

Reply:
I deleted it because result was never used as a string but maybe I missed something

Reply:
When `resultIsReject` is `true` (promise.catch() or async try/catch rejection), then `result` is a `string` corresponding to the error message. Otherwise `result` is a proper `THttpGetOpdsResultView` object (promise.then() resolve).